index.js

```diff
@@ -105,7 +170,21 @@ function findById() {}
  *  filterByGenre(movies, "Horror")
  *  //> []
  */
-function filterByGenre() {}
+function filterByGenre(movies, genre) {
+
+  let matches = [];
+
+  for(let i = 0; i < movies.length; i++){
+    let lowParam = genre.toLocaleLowerCase();
+    let lowGenre = movies[i].genre.toLocaleLowerCase().split(', ');
+    for(let j = 0; j < lowGenre.length; j++){
```

Comment:
While this works a nested loop gets a little messy to read.  How can we refactor this with `lowGenre.includes()`?